consortia/environments/values-dev.yaml

```diff
@@ -152,7 +152,11 @@ backend:
         - name: "HEALTHCHECKS__0__TAGS__2"
           value: "provisioningdb"
     onboardingServiceProvider:
-      encryptionKey: "<path:portal/data/dev/administration#onboardingserviceprovider-encryption-key>"
+      encryptionConfigs:
```

Comment:
Please provide a description about what to consider when upgrading from the previous encryption key, here as a comment and in the version upgrade file in portal-assets repository.

Reply:
see https://github.com/eclipse-tractusx/portal-assets/pull/279